executor/runtime/docker/docker_env.go

```diff
@@ -18,7 +18,7 @@ const envFileTemplateStr = `# This file was autogenerated by the titus executor
 export {{ $key }}={{ $val | escape_sq }}
 {{ end -}}
 
-{{ if (.ImageEnv | len) gt 0 }}
+{{ if .ImageEnv }}
```

Comment:
Not sure what this change is about?

Reply:
likewise

Reply:
That test stopped working, some template error. So, I fixed it. I believe this check is equivalent.